cavern/Cavern.properties

```diff
@@ -8,7 +8,6 @@ VOS_FILESYSTEM_ROOT = /cavern
 # 
 SSHFS_SERVER_BASE = proto.canfar.net:/
```

Comment:
I know this is just an example file, but could you please change this to ws-uv.canfar.net so that it looks up-to-date?

Reply:
Done